### ISSUE TITLE
Feature: Add option to make fonts bigger

### ISSUE THREAD

**Issue by adrian7** [docker/kitematic#1316]
Would really like an option in the UI to be able to choose/make fonts larger.

Thanks.

**Comment by FrenchBen** [comment]
Is there a specific UI area that would need larger font? Everything seems pretty big to me.

**Comment by adrian7** [comment]
The image descriptions is written in a small font size also the container logs.
Thanks.

**Comment by FrenchBen** [comment]
The description font is going to be tough, as a lot of the layout depends on the size of those boxes. Upping the description font size may throw off some of the boxes. You can always click the 3 `ooo` and click the 'View on Docker Hub' link.

Getting the log fonts 'larger' shouldn't be too hard.

**Comment by aquascaper** [comment]
I'd be happy to work on this!

**Comment by aquascaper** [comment]
You can check out the pull request here: 
https://github.com/docker/kitematic/pull/1325

**Comment by FrenchBen** [comment]
:+1: @aquascaper I'll checkout the PR. Thanks for the work

**Comment by aquascaper** [comment]
@FrenchBen Thanks!!! I'm looking forward to your feedback!

**Comment by rkazak** [comment]
Hi there are 3 exp/beginner tagged issues are they still in play?

**Comment by ceagan** [comment]
+1 This is a highly desirable feature especially for those of us with high-DPI monitors.

**Comment by FrenchBen** [comment]
@ceagan I agree - there were some comments added to the PR but haven't seen changes since then.

**Comment by aquascaper** [comment]
I made this PR https://github.com/docker/kitematic/pull/1479 to address the font size in the container log

**Comment by cockroachzl** [comment]
+1, the font on the container sidebar looks small.

**Comment by adrido** [comment]
Is there still some movement on this? Especially on a high-DPI screen the font gets unreadable small.
As a workaround I had to change  the compatibility mode to "Windows Vista" 😆 to get at least washy but readable text.

**Comment by FrenchBen** [comment]
@adrido The PR is still open with some changes needed. If you have time to help make it better, please contribute :)

**Comment by joejordan** [comment]
Came here looking for a a way to increase the font size in Kitematic. Fonts across the application could use a boost, but the log file font is especially small on my 20" 1080p screen on Windows 10.

**Comment by hellonearthis** [comment]
I was looking at the log file font sizing changing its font size from 10px to 12px looks much nicer.
Changing font-size in container-home.less details-panel.home . content .mini-logs .widget .logs works but it also changes the search bar font sizing.

The font weight could also be boosted but I've not got that far as I've only just started looking at the source.

Changing the colours for better contrasts also helps make the fonts more visible as the contrast is better.
The 6 gray scales is not good when it comes to visibility especially for people with contrast issues.  Adding alternative colour styles in the preferences would help ie default, dark, alt and high contrast.
[guidance, design and development contrast and the use of colour.](https://webtoolkit.govt.nz/guidance/design-and-development/contrast-and-the-use-of-colour/)

**Comment by ArianeLu** [comment]
The best way to resolve this issue is upgrade electron to latest version, nice compatibility with Hi-DPI monitor

**Comment by FrenchBen** [comment]
@ArianeLu Agreed, although that's also a full revamp of a lot of underlying functionality as well, which isn't a small change.
